lib/internal/process/stdio.js

```diff
@@ -8,26 +8,64 @@ function setupStdio() {
   function getStdout() {
     if (stdout) return stdout;
     stdout = createWritableStdioStream(1);
+
+    Object.defineProperty(stdout._writableState, 'ended', {
+      set() {},
+      get() {
+        return false;
+      }
+    });
+    // XXX(Fishrock123): This should work I think, but it causes
+    // problems with some child-process related tests.
+    //
+    // Object.defineProperty(stdout, 'destroyed', {
+    //   set() {},
+    //   get() {
+    //     return true;
+    //   }
+    // });
```

Comment:
@nodejs/streams this affected some other tests but I wasn't able to tell why.

```
=== release test-http-chunk-problem ===                                        
Path: parallel/test-http-chunk-problem
assert.js:85
  throw new assert.AssertionError({
  ^
AssertionError: '8c206a1a87599f532ce68675536f0b1546900d7a' == 'bafcbe972b6e69cd415ded38cb995f1bca983929'
    at cp.exec (/Users/Jeremiah/Documents/node/test/parallel/test-http-chunk-problem.js:55:20)
    at ChildProcess.exithandler (child_process.js:202:7)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:877:16)
    at Socket.<anonymous> (internal/child_process.js:334:11)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at Pipe._handle.close [as _onclose] (net.js:501:12)
```

&

```
=== release test-stdin-pipe-large ===                                          
Path: parallel/test-stdin-pipe-large
assert.js:85
  throw new assert.AssertionError({
  ^
AssertionError: 131072 === 1048576
    at Socket.child.stdout.on.common.mustCall (/Users/Jeremiah/Documents/node/test/parallel/test-stdin-pipe-large.js:22:10)
    at Socket.<anonymous> (/Users/Jeremiah/Documents/node/test/common.js:421:15)
    at emitNone (events.js:91:20)
    at Socket.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```